boards/arm/cc3220sf_launchxl/doc/cc3220sf_launchxl.rst

```diff
@@ -150,65 +146,70 @@ Prerequisites:
    Note that the emulation package install may place the xdsdfu utility
    in <install_dir>/ccs_base/common/uscif/xds110/.
 
-#. Ensure CONFIG_XIP=y is set.
+#. Switch Jumper SOP[2..0] (J15) back to [001].
+
+   Remove power from the board (disconnect USB cable) before switching jumpers.
+
+#. Install TI OpenOCD
 
-   Add a 'CONFIG_XIP=y' line to the project's prj.conf file.
+   Clone the TI OpenOCD git repository from: http://git.ti.com/sdo-emu/openocd.
+   Follow the instructions in the Release Notes in that repository to build
+   and install.
+
+   Since the default TI OpenOCD installation is /usr/local/bin/,
+   and /usr/local/share/, you may want to backup any current openocd
+   installations there.
+   If you decide to change the default installation location, also update
+   the OPENOCD path variable in :file:`boards/arm/cc3220sf_launchxl/board.cmake`.
+
+#. Ensure CONFIG_XIP=y (default) is set.
 
    This locates the program into flash, and sets CONFIG_CC3220SF_DEBUG=y,
    which prepends a debug header enabling the flash to persist over
    subsequent reboots, bypassing the bootloader flash signature
    verification.
 
-   See Section of the 21.10 of the `CC3220 TRM`_ for details on the
-   secure flash boot process.
+   See Section 21.10 "Debugging Flash User Application Using JTAG" of the
+   `CC3220 TRM`_ for details on the secure flash boot process.
 
-Flashing Command:
------------------
 
-Once the above prerequisites are met, use the UniFlash command line tool
-to flash the Zephyr image:
+Once the above prerequisites are met, applications for the ``_cc3220sf_launchxl``
+board can be built, flashed and debugged with openocd and gdb per the Zephyr
```

Comment:
``built, flashed, and debugged``  (missing comma)

Reply:
ah, the Oxford comma ;-)
OK, thanks for review.